openedx/core/djangoapps/user_api/management/commands/sync_hubspot_contacts.py

```diff
@@ -41,57 +40,43 @@ def _get_hubspot_enabled_sites(self):
         ]
         return hubspot_sites
 
-    def _get_last_synced_contact_email(self, site_conf):
+    def _get_users_queryset(self, initial_days):
         """
-        Returns: last synced contact email for given site
-
+        initial_days: numbers of days to go back from today
+        :return: users queryset
         """
-        api_key = site_conf.get_value('HUBSPOT_API_KEY')
-        last_contact_email = None
-        client = EdxRestApiClient(urlparse.urljoin(HUBSPOT_API_BASE_URL, 'contacts/v1/lists/all/contacts'))
-        try:
-            response = client.recent.get(hapikey=api_key, count=1, property='email')
-            if 'contacts' in response:
-                for contact in response['contacts']:
-                    last_contact_email = contact.get('properties').get('email').get('value')
-
-        except (HttpClientError, HttpServerError) as ex:
-            message = u'An error occurred while getting recent contact for site {domain}, {message}'.format(
-                domain=site_conf.site.domain, message=ex.message
-            )
-            self.stderr.write(message)
-        return last_contact_email
+        start_date = datetime.now().date() - timedelta(initial_days)
+        end_date = datetime.now().date() - timedelta(1)
+        if start_date == end_date:
```

Comment:
nit: I think we don't need different operations incase `start_date` and `end_date` are same.

Reply:
@ziafazal You mean it could be like this ?
```
start_date = datetime.now().date() - timedelta(initial_days)
        end_date = datetime.now().date() - timedelta(1)
        if start_date == end_date:
            self.stdout.write(u'Getting users on={date}'.format(date=start_date))
        else:
            self.stdout.write(u'Getting users from {start} to {end}'.format(start=start_date, end=end_date))
        return User.objects.filter(date_joined__date__gte=start_date, date_joined__date__lte=end_date)
```